composer.json

```diff
@@ -55,7 +55,8 @@
         "psr-4": {
             "ProxyManagerBench\\": "tests/ProxyManagerBench",
             "ProxyManagerTest\\": "tests/ProxyManagerTest",
-            "ProxyManagerTestAsset\\": "tests/ProxyManagerTestAsset"
+            "ProxyManagerTestAsset\\": "tests/ProxyManagerTestAsset",
+            "Zend\\": "tests/Stubbed/Zend"
```

Comment:
Let's be more specific about the `Zend\\` namespace here - we don't override all of `Zend\\` after all

Reply:
Fixed this one as well :)